workflow/rocoto/gfs_tasks.py

```diff
@@ -1490,7 +1502,7 @@ def gempakmeta(self):
 
     def gempakmetancdc(self):
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.cdump}gempak'}
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}gempak'}
```

Comment:
Same comment as for the gempakmeta job, no change was made for the gempakmetancdc job.

Reply:
Same response. These are the dependencies on the `gempak` job that _was_ changed.